src/api/component.js

```diff
@@ -125,11 +128,12 @@ export default class extends HTMLElement {
     Object.defineProperty(this, 'observedAttributes', { configurable: true, value });
   }
 
+  // returns inherited super props overwritten with this Component props
   static get props () {
-    return {};
+    return assign({}, super.props, this[$ctor_props]);
   }
   static set props (value) {
-    Object.defineProperty(this, 'props', { configurable: true, value });
+    this[$ctor_props] = value;
```

Comment:
I was getting stack size errors in IE11 if I didn't use `Object.defineProperty()`.